### PR TITLE
Add doc for enabling RBAC with Redpanda Operator

### PR DESCRIPTION
**Pull request by JakeSCahill**

### CODE REVIEW

File: charts/redpanda/values.yaml [hunk 1]
```diff
@@ -704,6 +704,9 @@ serviceAccount:
 # -- Role Based Access Control.
 rbac:
   # -- Enable for features that need extra privileges.
+  # If you use the Redpanda Operator,
+  # you must deploy it with the `--set rbac.createRPKBundleCRs=true` flag
+  # to give it the required ClusterRoles.
```
Comment (joejulian) [inline charts/redpanda/values.yaml:709]: What if I'm not allowed access to ClusterRoles?

Comment (JakeSCahill) [inline charts/redpanda/values.yaml:709]:
```suggestion
  # The Redpanda Operator does not include extra privileges by default.
  # If you intend to deploy a Redpanda resource with RBAC enabled,
  # ensure that you first deploy the Redpanda Operator with `rbac.createRPKBundleCRs` set to `true`.
```

Comment (JakeSCahill) [reply charts/redpanda/values.yaml:709]: @joejulian is this clearer?

Comment (joejulian) [reply charts/redpanda/values.yaml:709]: Much! Thanks!

**Review by joejulian — APPROVED** [review]

**Review by RafalKorepta — APPROVED** [review]

**Pull request merged**